dominion/static/js/profile_map_mapit.js

```diff
@@ -1,86 +1,181 @@
-// extend the default Wazimap ProfileMaps object to add mapit support
-
-var BaseProfileMaps = ProfileMaps;
-ProfileMaps = function() {
+var ProfileMaps = function() {
```

Comment:
Maybe we should add a story to turn [these](https://github.com/CodeForAfrica/HURUmap-apps/pull/135/files#diff-28f11c7b5f6af8ab955cf76000148291R5) mapit js files into react components.